engine/src/core/graphics/rendering/shaders/shader.h

```diff
@@ -0,0 +1,66 @@
+//
+// Created by blinku on 28.05.19.
+//
+
+#pragma once
+
+#include "unordered_map"
```

Comment:
for standard library <> and I don't see why we need this includes in this .h file (move this to cpp).